packages/shared/src/hooks/useEnableNotification.ts

```diff
@@ -92,17 +78,7 @@ export const useEnableNotification = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [shouldShowCta]);
 
-  useEffect(() => {
-    return () => {
-      if (onAcceptedPermissionJustNow) {
-        onAcceptedPermissionJustNow(false);
-      }
-    };
-  }, [onAcceptedPermissionJustNow]);
```

Comment:
This one is replaced elsewhere or how do we prompt this?

Reply:
This one is to ensure the banner is removed when the user moves out of the page/component. By using `useQuery` with a function that returns a default value false, this automatically happens.

Reply:
Ah cool got it, yeah just wanted to ensure it still happens on UI end.